src/ExportAPI.jl

```diff
@@ -3,9 +3,10 @@
 
 # DFG SpecialDefinitions
 export AbstractDFG,
-  InMemDFGType,
   getSolverParams,
   LightDFG,
+  GraphsDFG,
```

Comment:
should probably move export of LightDFg to Deprecated.jl

Reply:
Ok, sounds good, will update